apps/passport-client/pages/index.tsx

```diff
@@ -199,6 +281,9 @@ async function loadInitialState(): Promise<AppState> {
   const pcds = await loadPCDs();
   const encryptionKey = loadEncryptionKey();
   const subscriptions = await loadSubscriptions();
+  const offlineTickets = loadOfflineTickets();
+  const checkedOfflineInDevconnectTickets =
```

Comment:
Should be `checkedInOfflineDevconnectTickets`? "checked offline in" doesn't seem grammatical.

Reply:
done